src/storage-preview/azext_storage_preview/__init__.py

```diff
@@ -110,12 +109,12 @@ def register_common_storage_account_options(self):
         from ._validators import validate_encryption_services
 
         t_access_tier, t_sku_name, t_encryption_services = self.command_loader.get_models(
-            'AccessTier', 'SkuName', 'EncryptionServices', resource_type=CUSTOM_MGMT_STORAGE)
+            'AccessTier', 'SkuName', 'EncryptionServices', resource_type=CUSTOM_MGMT_PREVIEW_STORAGE)
 
         self.argument('https_only', help='Allows https traffic only to storage service.',
                       arg_type=get_three_state_flag())
         self.argument('sku', help='The storage account SKU.', arg_type=get_enum_type(t_sku_name))
-        self.argument('assign_identity', action='store_true', resource_type=CUSTOM_MGMT_STORAGE,
+        self.argument('assign_identity', action='store_true', resource_type=CUSTOM_MGMT_PREVIEW_STORAGE,
                       min_api='2017-06-01',
```

Comment:
seems `min_api` should also change

Reply:
May I know the reason to change min_api here?
From my perspective, the feature could be supported after 2017-06-01 in rest and the min api for the parameter is defined to 2017-06-01 because it is consistent with actual rest support. 
Resource type is only used to specify the api version used here.